assets/react/v3/entries/course-builder/components/modals/Question.tsx

```diff
@@ -0,0 +1,191 @@
+import SVGIcon from '@Atoms/SVGIcon';
+import { borderRadius, colorTokens, shadow, spacing } from '@Config/styles';
+import { typography } from '@Config/typography';
+import { QuizQuestion, QuizQuestionType } from '@CourseBuilderServices/quiz';
+import ThreeDots from '@Molecules/ThreeDots';
+import { IconCollection, isDefined } from '@Utils/types';
+import { css } from '@emotion/react';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { animateLayoutChanges } from '@Utils/dndkit';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+interface QuestionProps {
+  question: QuizQuestion;
+  index: number;
+  selectedQuestionId: number | null;
+  setSelectedQuestionId: (id: number | null) => void;
+}
+
+const questionTypeIconMap: Record<QuizQuestionType, IconCollection> = {
+  'true-false': 'quizTrueFalse',
+  'single-choice': 'quizSingleChoice',
+  'multiple-choice': 'quizMultiChoice',
+  'open-ended': 'quizEssay',
+  'fill-in-the-blanks': 'quizFillInTheBlanks',
+  'short-answer': 'quizShortAnswer',
+  matching: 'quizMatching',
+  'image-matching': 'quizImageMatching',
+  'image-answering': 'quizImageAnswer',
+  ordering: 'quizOrdering',
+};
+
+export const Question = ({ question, index, selectedQuestionId, setSelectedQuestionId }: QuestionProps) => {
+  const [isActive, setIsActive] = useState<boolean>(index === 0);
+  const { attributes, listeners, setNodeRef, transform, transition, isDragging } = useSortable({
+    id: question.ID,
+    animateLayoutChanges,
+  });
+
+  const wrapperRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const handleOutsideClick = (event: MouseEvent) => {
+      if (isDefined(wrapperRef.current) && !wrapperRef.current.contains(event.target as HTMLDivElement)) {
+        setIsActive(false);
+      }
+    };
+
+    document.addEventListener('click', handleOutsideClick);
+
+    return () => document.removeEventListener('click', handleOutsideClick);
+  }, []);
```

Comment:
This is not required I believe. We do not want to deactivate the item on outside click.